v2/recipes/state-machine-transitions.md

```diff
@@ -0,0 +1,142 @@
+# State Machine Transitions
+
+## Finding Transitions
+Transition information can be found in a `{module}.workflows.yml`.
+
+Example from commerce_order:
+
+```yaml
+order_default:
+  id: order_default
+  group: commerce_order
+  label: 'Default'
+  states:
+    draft:
+      label: Draft
+    completed:
+      label: Completed
+    canceled:
+      label: Canceled
+  transitions:
+    place:
+      label: 'Place order'
+      from: [draft]
+      to: completed
+    cancel:
+      label: 'Cancel order'
+      from: [draft]
+      to:   canceled
+
+order_default_validation:
```

Comment:
Remove this + below, we can just keep the reference to the default `order_default`